### PR TITLE
webui: upgrade to patternfly v5

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Comment by KKoukiou** [comment]
View pixel test diffs here https://cockpit-logs.us-east-1.linodeobjects.com/pull-5054-20230817-144107-efc66430-fedora-rawhide-boot/log.html

File: ui/webui/src/components/storage/InstallationMethod.jsx [hunk 1]
```diff
@@ -84,10 +91,206 @@
...
+        const enabledMenuItems = selectOptions.filter((menuItem) => !menuItem.isDisabled);
+        const [firstMenuItem] = enabledMenuItems;
+        const focusedItem = focusedItemIndex ? enabledMenuItems[focusedItemIndex] : firstMenuItem;
+
+        switch (event.key) {
+        // Select the first available option
+        case "Enter":
+            if (!isOpen) {
+                setIsOpen((prevIsOpen) => !prevIsOpen);
+            } else if (isOpen && focusedItem.name !== "no results") {
```
[the remark this answers is not in the document]
Comment (KKoukiou) [reply ui/webui/src/components/storage/InstallationMethod.jsx]: Fixed.

**Review by garrett — APPROVED** [review]
I see a couple of minor things that concern me, but I don't think they're PF5 related.

1. The sidebar. We probably should make it larger anyway, when the screen size allows for it:

> Ugh, yeah. Perhaps we want to override the width for that anyway... if there's a larger screen, then allow for it to be a bit wider. Like using width: max(--pf-5-whatever-it-is, 20vw) or something

2. The lack of appropriate and consistent vertical spacing on the method list of radio items:

> this might not be a change, but it looks like there isn't enough vertical spacing between the radio items, especially between the last two (which could be an optical illusion; I haven't measured yet) [https://cockpit-logs.us-east-1.linodeobjects.com/pull-5054-20230817-144107-efc66430-fedora-rawhid[…]isks-fedora-rawhide-boot-127.0.0.2-2201-FAIL.png](https://cockpit-logs.us-east-1.linodeobjects.com/pull-5054-20230817-144107-efc66430-fedora-rawhide-boot/TestStorageMountPoints-testMultipleDisks-fedora-rawhide-boot-127.0.0.2-2201-FAIL.png)
> 
> the text is too jumbled up together; it should be more like the mockup... this might not be a PF5 blocker though
> 
> I don't think I see anything that would block the PF5 PR; there are a few optimizations we want to do after though (enlarge the sidebar when there's room to do so, make sure there's proper space between the items on the method list of radio items)

I'd say this PF5 PR is fine and we can address these in minor fixups afterward.

**Review by VladimirSlavik — COMMENTED** [review]
Changes mostly LGTM.

Does it make sense to bandwagon the last two code commits as part of this? I don't think it's part of the migration.

**Comment by KKoukiou** [comment]
> Changes mostly LGTM.
> 
> Does it make sense to bandwagon the last two code commits as part of this? I don't think it's part of the migration.

Indeed - these are seperate I can adjust.

**Review by VladimirSlavik — COMMENTED** [review]
Thanks, as it is right now, I'd ack if I trusted myself with JS.

**Review by rvykydal — APPROVED** [review]
Looks good to me AFAICT.

**Pull request merged**